go/runtime/scheduling/api/api.go

```diff
@@ -6,14 +6,14 @@ import (
 	"github.com/oasisprotocol/oasis-core/go/runtime/transaction"
 )
 
-// Algorithm defines an algorithm for scheduling incoming transaction.
-type Algorithm interface {
-	// Initialize initializes the internal transaction scheduler state.
-	// Algorithm should use the provided transaction dispatcher to dispatch
-	// scheduled transactions.
+// Scheduler defines an algorithm for scheduling incoming transaction.
```

Comment:
```suggestion
// Scheduler defines an algorithm for scheduling incoming transactions.
```

Reply:
Was this reverted, I remember this was already fixed :-)

Reply:
Yeah me to :thinking: